ui/Document.tsx

```diff
@@ -29,7 +31,12 @@ export function Document({ html, css, scripts }: DocumentProps) {
               dangerouslySetInnerHTML={content ? { __html: content } : undefined}
             />
           ))}
-        <script src={'/ui.js'} />
+        <script
+          dangerouslySetInnerHTML={{
+            __html: `window.__APOLLO_STATE__=${JSON.stringify(state).replace(/</g, '\\u003c')};`,
```

Comment:
What's up with this `replace`?

Reply:
Unsure. It's from the apollo docs. Looks like replacing '</> with the HTML character code.